data/css/all-aboard.css

```diff
@@ -219,6 +219,12 @@ a:focus {
     display: inline-block;
 }
 
+.token-received-msg {
```

Comment:
@schalkneethling can you make this font normal instead of bold and slightly smaller? probaby 1.0rem instead of 1.2.